src/sail_json_backend/json.ml

```diff
@@ -600,9 +621,12 @@ let defs { defs; _ } =
     | DEF_aux (DEF_mapdef (MD_aux (MD_mapping (i, _, ml), _)), _) ->
         debug_print "DEF_mapdef";
         List.iter (parse_mapcl i) ml
+    | DEF_aux (DEF_register (DEC_aux (DEC_reg (atyp, id, _), _)), _) ->
+        let reg_name = string_of_id id in
+        let reg_type = string_of_typ atyp in
+        Hashtbl.add registers reg_name reg_type
     | _ -> debug_print ~printer:prerr_string ""
   ) defs;
-
```

Comment:
lost a blank line here. Add that back, please.

Reply:
Is it added back? I can't tell due to the UI, I see the blank line [here](https://github.com/AbhinavMir/sail/blob/953f3f34bdf7ba9e3ac314822a5a7e6ad0a83aba/src/sail_json_backend/json.ml#L516).

Reply:
I see a blank line. The best was to look is to go to the "Files changed" tab.

However, now that I'm looking again, the blank line contains 2 spaces, thus is different than the original line. Remove those 2 spaces, and this should all disappear.